### ISSUE TITLE
Validator exit function

### ISSUE THREAD

**Issue by paulhauner** [sigp/lighthouse#106]
## !! WAITING ON RELEASE CANDIDATE !!

This issue is presently on-hold until the specification is stable.

## Description

The [Routine for exiting a validator](https://github.com/ethereum/eth2.0-specs/blob/master/specs/core/0_beacon-chain.md#routine-for-exiting-a-validator) section of the spec describes the method for exiting a an active validator.

## Tasks

- Implement  the `exit_validator` function.

## Blocking

This is blocked on:

- #81

**Comment by paulhauner** [comment]
This task is on-ice. Initially we're going to target a static validator set, therefore we will not need an `exit` function.

There's no good reason to build this component at this point.

**Comment by paulhauner** [comment]
Closing in favor of #182
